docs/support.md

```diff
@@ -1,6 +1,6 @@
 # Support
 
 Best way to ask general questions about documentation or particular use cases is to [create a GitHub issue](https://github.com/cirruslabs/cirrus-ci-com/issues/new).
-Support team will answer it by replying or updating documentation.
+Our support team will answer it by replying or updating documentation. They tend to respond within 48 hours.
 
-For any other questions please email us at [support@cirruslabs.org](mailto:support+ci@cirruslabs.org).
+For any other questions please email us at [support+ci@cirruslabs.org](mailto:support+ci@cirruslabs.org).
```

Comment:
This is intentionally to have a separate label. `support@cirruslabs.org` also works though.